### ISSUE TITLE
[Bug]: netclient 0.12

### ISSUE THREAD

**Issue by ahmaddxb** [gravitl/netmaker#916]
### What happened?

Docker install
error
2022/03/18 12:06:54 [netclient] error removing profile from /etc/hosts 'default' profile should not be handled by hostctl

### Version

v0.12.0

### What OS are you using?

Linux

### Relevant log output

```shell
netclient] joining network

2022/03/18 12:06:45 [netclient] success

2022/03/18 12:06:45 [netclient] ALREADY_INSTALLED. Netclient appears to already be installed for default. To re-install, please remove by executing 'sudo netclient leave -n default'. Then re-run the install command.

[netclient] Starting netclient daemon

2022/03/18 12:06:45 [netclient] pulling latest config for PFIep0SZ

2022/03/18 12:06:47 [netclient] waiting for interface...

2022/03/18 12:06:47 [netclient] interface ready - netclient.. ENGAGE

2022/03/18 12:06:49 [netclient] pulling latest config for default

2022/03/18 12:06:50 [netclient] waiting for interface...

2022/03/18 12:06:50 [netclient] interface ready - netclient.. ENGAGE

2022/03/18 12:06:53 [netclient] started comms network daemon, PFIep0SZ

2022/03/18 12:06:53 [netclient] netclient daemon started for network: PFIep0SZ

2022/03/18 12:06:53 [netclient] received peer update for node last-ornery PFIep0SZ

2022/03/18 12:06:53 [netclient] received message to update node instance-20220222-1222

2022/03/18 12:06:54 [netclient] received peer update for node instance-20220222-1222 default

2022/03/18 12:06:54 [netclient] error removing profile from /etc/hosts 'default' profile should not be handled by hostctl
```

### Contributing guidelines

- [X] Yes, I did.

**Comment by mattkasun** [comment]
This is a limitation of  the library used to control /etc/hosts.  The original contents of /etc/hosts are allocated to the profile "default".  Netclient uses the network name as the profile with the library.  As "default" is reserved, private dns cannot be used with a network named default.

**Comment by ahmaddxb** [comment]
I did a clean install of netmaker and don't receive this error anymore.

**Comment by mattkasun** [comment]
fixed with PR #918

**Comment by ahmaddxb** [comment]
I have this error

2022/03/22 11:46:20 [netclient] received peer update for node localhost vlan
2022/03/22 11:46:20 [netclient] error removing profile from /etc/hosts unknown profile name
